.github/workflows/test.yml

```diff
@@ -14,29 +14,21 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, windows-latest ]
-        node: [ 12, 14, 16 ]
+        node: [ 14, 16 ]
```

Comment:
We dropped Node 12 from our testing CI since it's not supported by Vitest

This wasn't caught sooner since we were hard-coding `node-version` to `14` instead of using `${{matrix.node}}`